projects/media-viewer/src/lib/annotations/annotation-set/annotation/comment/comment.component.ts

```diff
@@ -35,6 +36,34 @@ export class CommentComponent {
     this.fullComment = comment.content;
   }
 
+  @Input()
+  set selected(selected: boolean) {
+    this._selected = selected;
+  }
+
+  get selected() {
+    return this._selected || Boolean(!this.author && this.fullComment);
+  }
+
+  @Input()
+  set editable(editable: boolean) {
```

Comment:
I'm confused, why are we using set on these?

Reply:
same as above, to handle draft comments